aiida/orm/computer.py

```diff
@@ -703,6 +703,62 @@ def get_scheduler(self):
             raise exceptions.ConfigurationError('No scheduler found for {} [type {}], message: {}'.format(
                 self.name, self.get_scheduler_type(), exc))
 
+    def configure(self, user=None, **kwargs):
+        """
+        Configure a computer for a user with valid auth params passed via kwargs
+
+        :param user: the user to configure the computer for
+        :kwargs: the configuration keywords with corresponding values
+        :return: the authinfo object for the configured user
+        :rtype: :class:`aiida.orm.AuthInfo`
+        """
+        from aiida.orm.backend import construct_backend
+
+        transport_cls = self.get_transport_class()
+        backend = construct_backend()
```

Comment:
Since this is now a member method of the `Computer` instance, we should get the `backend` from its own attribute and not construct it again
```suggestion
        backend = self.backend
```